src/EditorFeatures/Next/IntelliSense/Completion/Presentation/Roslyn15CompletionSet.cs

```diff
@@ -1,104 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Globalization;
-using System.Linq;
-using Microsoft.CodeAnalysis.Completion;
-using Microsoft.CodeAnalysis.Text.Shared.Extensions;
-using Microsoft.VisualStudio.Language.Intellisense;
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Editor;
-
-namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Presentation
-{
-    /// <summary>
-    /// See comment on <see cref="VisualStudio15CompletionSet"/> for an explanation of how
-    /// these types fit together and where code should go in them.
-    ///
-    /// This class is where all code that would normally be in our derived type of 
-    /// <see cref="CompletionSet2"/> should go.  <see cref="VisualStudio15CompletionSet"/> 
-    /// should then just forward all calls to us and we can share all our Dev14 logic with 
-    /// <see cref="Roslyn14CompletionSet"/>.
-    /// </summary>
-    internal class Roslyn15CompletionSet : Roslyn14CompletionSet
```

Comment:
@CyrusNajmabadi Will that help? I think the bigger problem here is me flattening the hierarchy, so Git lost track of what was going on.